hypertrace-gradle-java-convention-plugin/src/main/java/org/hypertrace/gradle/java/convention/JavaConventionPlugin.java

```diff
@@ -46,6 +47,15 @@ private void configureCompilerRelease(Project target) {
   }
 
   private void configureToolchain(Project target) {
+    if (javaConventionExtension(target)
```

Comment:
Since this compatibility is (as far as we know) only in the codestyle plugin, wouldn't it make more sense to warn there?

Reply:
In other words - if you use just this plugin without codestyle, you won't have an issue. If you use codestyle and have a higher java toolchain (with or without this plugin), you might.